tools/mpy_ld.py

```diff
@@ -1465,6 +1490,7 @@ def main():
     cmd_parser.add_argument("--arch", default="x64", help="architecture")
     cmd_parser.add_argument("--preprocess", action="store_true", help="preprocess source files")
     cmd_parser.add_argument("--qstrs", default=None, help="file defining additional qstrs")
+    cmd_parser.add_argument("-l", dest="libs", action="append", help="Static .a libraries to link")
```

Comment:
Suggest adding a long form name `--libs` or `--lib` for this (as well as keeping `-l`).

And lowercase "static .a ..." to match the help of other commands.